packages/amplify_analytics_pinpoint/example/analysis_options.yaml

```diff
@@ -0,0 +1,5 @@
+include: package:amplify_lints/flutter_app.yaml
+
+linter:
+  rules:
+    avoid_print: false
```

Comment:
Nit: eof newline

I'm just pointing these out since GH is pointing them out. You can either ignore or just comb through these and add.

Reply:
Hmm I will add it to my files and I've enabled it to automatically be added when saving in Vs coe- however some of these are generated files so I'm not sure it makes sense to go through those.